prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -5,7 +5,7 @@ presubmits:
       branches:
         - master
       decorate: true
-      always_run: true
+      run_if_changed: '^prow/(cluster/|prowjobs/|config.yaml$|plugins.yaml$)'
```

Comment:
this job seems pretty cheap and most of this repo is yaml? 🤷‍♂ 
I continue to claim `run_if_changed` is a footgun to let changes slip through.

Reply:
/shrug
Switched back to `always_run: true`.